react/react2/react2-week1-ex2/app/form/page.jsx

```diff
@@ -0,0 +1,66 @@
+"use client";
+import React, { useRef } from "react";
+import TextField from "@mui/material/TextField";
+import Button from "@mui/material/Button";
+
+export default function Form() {
+  // Create refs for each input field
+  const firstNameRef = useRef(null);
+  const lastNameRef = useRef(null);
+  const emailRef = useRef(null);
+  const phoneRef = useRef(null);
+
+  // Function to handle key press and move focus to the next field
+  const handleKeyPress = (e, nextRef) => {
+    if (e.key === "Enter") {
+      // Focus on the next input field if Enter is pressed
+      nextRef.current.focus();
+    }
+  };
+
+  return (
+    <div
+      style={{
+        display: "flex",
+        flexDirection: "column",
+        gap: "16px",
+        maxWidth: "300px",
+        margin: "50px auto",
+      }}
+    >
+      {/* First Name Input */}
+      <TextField
+        sx={{ marginTop: "2rem" }}
+        label="First Name"
+        variant="outlined"
+        inputRef={firstNameRef}
+        onKeyPress={(e) => handleKeyPress(e, lastNameRef)} // Move focus to the next input
+      />
+
+      {/* Last Name Input */}
+      <TextField
+        label="Last Name"
+        variant="outlined"
+        inputRef={lastNameRef}
+        onKeyPress={(e) => handleKeyPress(e, emailRef)} // Move focus to the next input
+      />
+
+      {/* Email Input */}
+      <TextField
+        label="Email"
+        variant="outlined"
+        inputRef={emailRef}
+        onKeyPress={(e) => handleKeyPress(e, phoneRef)}
+      />
+      {/* Phone Input */}
+      <TextField
+        label="phone"
+        variant="outlined"
+        inputRef={phoneRef}
+        onKeyPress={(e) => handleKeyPress(e, firstNameRef)}
+      />
+
+      <Button variant="contained">Submit</Button>
```

Comment:
This button is not really "submit" button and it won't generate `submit` event when you click on it (which means there is no way you can get data grom this form and submit it anywhere): you should wrap inputs in `form` element and add `type=submit` attribute to your button

It would be necessary to implement the rest of task (I don't see any from it in here)

<img width="574" alt="image" src="https://github.com/user-attachments/assets/b3ce8c38-4c86-4d23-aa3a-d3241aeea845">

Reply:
My code disapeared somehow when I was playing around git I suppose :D I ll implement it again